workloads/templates/workload-fio-script-cm.yml.j2

```diff
@@ -0,0 +1,244 @@
+         echo "waiting on project {{ fiotest_basename }}0 to disappear ..."
+         sleep 10
+       done
+       echo "Project {{ fiotest_basename }}0 is deleted ... test finished"
+    elif [[ {{ fiotest_cleanup }} == "false" ]]; then
+       echo "Test is done, but project {{ fiotest_basename }}0 is not be deleted due to FIOTEST_CLAENUP=false"
+    fi
+  fiotest.sh: |
+    #!/bin/bash
+    cp /root/.kube/config /tmp/config
+    export KUBECONFIG=/tmp/config
+    # Not sure is this necessary - re-test
+    sed -i 's/\-o StrictHostKeyChecking=no/\-o StrictHostKeyChecking=no \-l default \-p 2022/g' /opt/pbench-agent/base
+    # to do - fix pbench-fio to support custom options
+    # open issue why getconf.py returns empty string in container
+    curl -o /opt/pbench-agent/bench-scripts/pbench-fio https://raw.githubusercontent.com/ekuric/openshift/master/iotest/pbench-fio
+    oc get pods -n {{ fiotest_basename }}0 -o wide | awk '{print $6}' |grep -v IP > /tmp/clients.txt
+    pbench-fio --test-types={{ fiotest_testtype }} --config="{{ fiotest_testtype }}-pods{{ fiotest_maxpods }}-storageclass-{{ fiotest_storageclass }}" \
+    --clients=$(cat /tmp/clients.txt | awk -vORS=, '{ print $1 }' | sed 's/,$/\n/') -b {{ fiotest_bs }} \
+    --samples {{ fiotest_samples }} --runtime {{ fiotest_runtime }}  --iodepth {{ fiotest_iodepth }} --direct {{ fiotest_direct }} \
+    --targets {{ fiotest_filename }} -s {{ fiotest_filesize }} 
+  fiotest.yml: |
+    provider: local
+    ClusterLoader:
+      cleanup: {{ fiotest_cleanup }}
+      projects:
+        - num: 1
+          basename: {{ fiotest_basename }}
+          ifexists: delete
+          templates:
+            - num: 1
+              file: /root/workload/pbench-ssh.yaml
+              parameters:
+              - SSH_AUTHORIZED_KEYS: {{ storage_fio_ssh_authorized_keys }}
+              - SSH_PRIVATE_KEY: {{ storage_fio_ssh_private_key }}
+              - SSH_PUBLIC_KEY: {{ storage_fio_ssh_public_key }}
+            - num: {{ fiotest_maxpods }}
+              file: fiotesttemplate.yaml
+              parameters:
+              - STORAGE_FIO_IMAGE: {{ storage_fio_test_image }}
+              - FIO_SSHD_PORT: {{ storage_fio_test_sshd_port }}
+      tuningsets:
+        - name: default
+          pods:
+            stepping:
+              stepsize: {{ fiotest_stepsize }}
+              pause: {{ fiotest_pause }}
+            ratelimit:
+                delay: 0
+  fiotesttemplate.yaml: |
+    kind: Template
+    apiVersion: v1
+    metadata:
+      name: fio
+      annotations:
+        description: A template for creating an fio pod with PVC
+        tags: fio,perf
+    labels:
+      name: fiotest-test
+    objects:
+    - apiVersion: v1
+      kind: PersistentVolumeClaim
+      metadata:
+        name: "${PVC_NAME}"
+        annotations:
+          volume.beta.kubernetes.io/storage-class: {{ fiotest_storageclass }}
```

Comment:
We should strive to use the oc process command "templating" when possible. (Ideally this reduces cognitive load with templating since there are multiple things templating here)